src/Core/Grid/Definition/Factory/OrderGridDefinitionFactory.php

```diff
@@ -195,6 +196,7 @@ protected function getColumns()
                     'field' => 'new',
                     'true_name' => $this->trans('Yes', [], 'Admin.Global'),
                     'false_name' => $this->trans('No', [], 'Admin.Global'),
+                    'clickable' => true,
```

Comment:
I think it's useless here since `DataColumn` is already clickable by default

Reply:
BooleanColumn default value for `clickable` is false